compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -30,10 +30,12 @@
 namespace
 {
 
-std::ostream &operator<<(std::ostream &os, const std::vector<int32_t> &vect)
+template <typename VectorType>
+std::enable_if_t<std::is_same<typename VectorType::value_type, int32_t>::value, std::ostream &>
+operator<<(std::ostream &os, const VectorType &vect)
```

Comment:
could you explain why use `enable_if_t` ?

Reply:
I changed this function to template with same rationale as in https://github.com/Samsung/ONE/pull/7901/files#r740778845. 

But this definition without `enable_if`  redefines `operator<<` definition for many other entities, e.g. `std::string`. So, I used `enable_if_t` here. 

--------

I think in this case better to just changes type of operator's argument, because this change looks confusing. Let's introduce tenplate impl later if needed, Thank you.

Reply:
Made as not template solution.